.gitattributes

```diff
@@ -0,0 +1,4 @@
+###############################################################################
+# Set default behavior to automatically normalize line endings.
+###############################################################################
+* text=auto
```

Comment:
What is this for? :)

Reply:
That's the tip from Bjørnar to fix so I don't get a tidal wave of lint errors for line-endings every time I open a file 😅

Reply:
Allright, nice that you got the problem fixed :)